config/can_config.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+typedef enum CANID {
+
+  // ERROR CAN IDs
+
+  SAFETY_ERROR = 0x100,
+  ARM_ERROR,
+  SCIENCE_ERROR,
+  GIMBTONOMY_ERROR,
```

Comment:
it looks like this branch hasnt been updated in a while. this file doesnt exist anymore. you should merge master into ur branch

Reply:
after merging I still see this file present, should I delete it from my branch?

Reply:
You should delete it yes, but @younesr1 are all these ids already in bridge repo?

Reply:
alright ill remove the file